### PR TITLE
Disabled autocomplete still displays its list

### PR DESCRIPTION
**Pull request by EstherDarkish**
Only display the current substation in the disabled autocomplete field in the voltage level modification dialog to prevent a mui/material bug from showing all the other options, even if the field is disabled.

### CODE REVIEW

**Review by AAJELLAL — COMMENTED** [review]
Strange behaviour: Even if the field looks disabled but, if you click multiple times, the drop down is shown with one value in it but as a solution at the moment it's OK.

File: src/components/dialogs/network-modifications/voltage-level/modification/voltage-level-modification-form.js [hunk 1]
```diff
@@ -107,7 +96,10 @@ const VoltageLevelModificationForm = ({
             //setting null programatically when freesolo is enable wont empty the field
             name={SUBSTATION_ID}
             label="SUBSTATION"
-            options={substations}
+            // Because of a mui/material bug, the disabled attribute do not work properly.
```
[the remark this answers is not in the document]
Comment (EstherDarkish) [reply src/components/dialogs/network-modifications/voltage-level/modification/voltage-level-modification-form.js:99]: In the future, the field should be editable, when Powsybl allows it.

**Comment by EstherDarkish** [comment]
> Strange behaviour: Even if the field looks disabled but, if you click multiple times, the drop down is shown with one value in it but as a solution at the moment it's OK.
Yes, that's the less ugly solution for the moment.

**Review by AAJELLAL — APPROVED** [review]
Code OK.
Test OK.

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Pull request merged**